tests/test_log.py

```diff
@@ -100,3 +105,62 @@ def test_extract_boolean_value(self):
         eis = log.find("EventInfo")
         for ei in eis:
             assert isinstance(ei.time_event_info, bool), "Expected ei.time_event_info to be bool"
+
+class TestNoLogFile:
+    """
+        Test invoking functions on the FMU if we have no logfile.
+        We do this from temporary directories to make sure that we have empty directories for the test.
+    """
+    @classmethod
+    def setup_class(cls):
+        cls.fmu_path = os.path.join(get_examples_folder(), 'files', 'FMUs', 'ME2.0', 'bouncingBall.fmu')
```

Comment:
Should this not be: cls.fmu_path = os.path.join(get_examples_folder(), 'files', 'FMUs', 'XML', 'ME2.0', 'bouncingBall.fmu') ?

Reply:
That would be the case if it was the folder 'files' in 'tests' but now this FMU is picked from the example folder instead and the path is slightly different there.